### ISSUE TITLE
Improve IU/UX when getting public shared URL

### ISSUE THREAD

**Issue by henriquebs12** [nextcloud/server#14184]
It would be nice if you return to the old style when sharing a file. Now you don't see the URL, only the tip 'Copied', but that's a terrible UX, because sometimes I click on it, and because I have lots of files, I need to wait for the message to be displayed, which takes about 2 seconds. If I click in the icon to share and move the cursor away, it sometimes will copy to the clipboard but sometimes not.
 
This is so terrible because I use this feature everyday, and I have to check every URL to see if it has updated to the clipboard. Sometimes it doesn't and I end up using the same URL in the wrong place.

Please, please, please, we want to see the URL, we want to make sure it is shared and visible. We don't want it to be worst then a previous update. 
Summing up:

- The old input, where you could see the text, and when you click the input the text is selected and copied to clipboard

**Comment by skjnldsv** [comment]
We could indeed use a link so it also allow a right click and copy to clipboard.
Though if you have to wait 2sec to have the data copied, this is probably the bigger issue there :/

**Comment by szaimen** [comment]
As this sounds like a nice feature, the requests for this are quite low. Currently there are no plans to implement such a feature. Thus I will close this ticket for now. This does not mean we don't want this feature, but it is simply not on our roadmap for the near future. If somebody wants to implement this feature nevertheless we are happy to assist and help out.

If you wish to have this feature implemented by the Nextcloud GmbH there is the option for consulting work on top of your Nextcloud Enterprise subscription to get your features implemented.
